config/initializers/ministerial_report.rb

```diff
@@ -0,0 +1,146 @@
+module MinisterialReport
```

Comment:
👌  great to have this centralised. We could split hairs about exactly where this file belongs but I think this is fine